parts/linux/cloud-init/artifacts/init-aks-custom-cloud-flatcar.sh

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash
+set -x
+mkdir -p /root/AzureCACertificates
+# http://168.63.129.16 is a constant for the host's wireserver endpoint
+certs=$(curl "http://168.63.129.16/machine?comp=acmspackage&type=cacertificates&ext=json")
```

Comment:
this seems like a lot of code duplication, could we possibly detect within the original scripts that we're running on a flatcar distro and pivot accordingly?